facefusion/processors/frame/modules/frame_enhancer.py

```diff
@@ -138,10 +125,61 @@ def post_process() -> None:
 		clear_content_analyser()
 
 
+def split_frame_into_tiles(frame: VisionFrame, tile_size : int, pad_size : int) -> Tuple[numpy.ndarray[Any, Any], Tuple[int, int, int]]:
+    pad_size_bottom = pad_size + tile_size - frame.shape[0] % tile_size
+    pad_size_right = pad_size + tile_size - frame.shape[1] % tile_size
+    pad_frame = cv2.copyMakeBorder(frame, pad_size, pad_size_bottom, pad_size, pad_size_right, cv2.BORDER_REPLICATE)
+    tiles = []
+    for row in range(pad_size, pad_frame.shape[0] - pad_size, tile_size):
+        for column in range(pad_size, pad_frame.shape[1] - pad_size, tile_size):
+            top = row - pad_size
+            bottom = row + pad_size + tile_size
+            left = column - pad_size
+            right = column + pad_size + tile_size
+            tiles.append(pad_frame[top : bottom, left : right, :])
+    return numpy.array(tiles), pad_frame.shape
+
+
+def stitch_tiles_into_frame(tiles : numpy.ndarray[Any, Any], pad_frame_shape : Tuple[int, int, int], target_shape : Tuple[int, int, int], pad_size: int) -> VisionFrame:
+    tiles = tiles[:, pad_size:-pad_size, pad_size:-pad_size, :]
+    tile_height, tile_width, _ = tiles.shape[1:]
+    tiles_per_row = min(pad_frame_shape[1] // tile_width, len(tiles))
+    frame = numpy.zeros(pad_frame_shape)
+    for index, tile in enumerate(tiles):
+        top = (index // tiles_per_row) * tile_height
+        bottom = top + tile_height
+        left = (index % tiles_per_row) * tile_width
+        right = left + tile_width
+        frame[top : bottom, left : right, :] = tile
+    frame = frame[:target_shape[0], :target_shape[1], :].astype(numpy.uint8)
+    return frame
+
+
 def enhance_frame(temp_vision_frame : VisionFrame) -> VisionFrame:
+	frame_processor = get_frame_processor()
+	pre_pad_size = 15
+	pad_size = 24
+	scale = get_options('model').get('scale')
+	tile_size = get_options('model').get('tile_size') - (2 * pad_size)
+	temp_vision_frame = cv2.copyMakeBorder(temp_vision_frame, pre_pad_size, pre_pad_size, pre_pad_size, pre_pad_size, cv2.BORDER_REFLECT)
```

Comment:
introduce methods that do a certain job... either they normalize or prepare or ...